src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final int side;
+
+    public Square(int side, String color) {
+        super(color);
+        this.side = side;
+    }
+
```

Comment:
Not everywhere there is an `override` annotation for this method. Add in all necessary classes.